### ISSUE TITLE
Features for new release

### ISSUE THREAD

**Issue by Hospes** [KevinTyrrell/FishingBot#15]
Hi Kevin,

Thanks for your work, it's pretty good bot. But I think you have to do small changes to UI/logic to make bot more accurate.

1. For example bot trying to cast "/cast Fishing" so it would work only for english clients of the game cause in my case (RU) it wouldn't  work. So I have to manually press "start fishing".
Solution: just add field on program UI where user can enter macros that launch finishing. In this case you will be language independent.

2. You implement hardcoded zone where software will searching bobber, my proposal to implement customizing are. Like you did for calibration process. So user will be able to select zone where bobber would be potentially placed.

3. Randomizing - bot shouldn't use constant values at all. To make it more "human like" you have to each time randomize timings. Delay between casting fishing and etc.

4. According to catching fish, right now it's pretty hard to make it working well. Maybe would be best solution to check sound? Like it done in UltimateFishing bot written on C# ?

All of this just proposals, I don't have time now for making all this features by myself right now. Maybe later I'll do some pull requests if you are okay with it.

**Comment by KevinTyrrell** [comment]
Thank you @Hospes for taking the time to write this. Let me address your issues.

1. I agree. When I wrote this originally, I did not expect foreign users to find and use it. I could not think of a nice way for my program to know what phrase to type on non-english clients. I also did not want to have it 'press the 1 key' or something similar, as I wanted it to be simple to setup. I will setup a way for users to modify how it begins fishing.

2. I agree. Allowing the user to drag the area in which the bobber scans should be implemented. I will do this.

3. [This is actually implemented just as you say](https://github.com/KevinTyrrell/FishingBot/blob/6d0dad972786ad0ec1924ba956ce4be68d3a530d/src/model/Angler.java#L99), however I offer the user no control over the random variance. I will allow the user to control this setting in the future.

4. The bot was actually designed for patch 1.12.1 of World of Warcraft. Much has changed since including how the graphics are drawn. The challenging texture of the water makes it more difficult for my program to spot the bobber. I will begin testing on both retail and older versions of World of Warcraft to make sure it performs well in both.

As much as help would be appreciated, the master branch will not compile currently, which is not a good practice on my part. I will return to this and completely finish my implementation, then I will address these issues.

Thank you so much for your report. I never expected my program to get hundreds and hundreds of users.
